### PR TITLE
Adds dynamic text to the homepage that updates with each refresh

### PR DESCRIPTION
**Pull request by nitinramadoss**
Uses code from DataServlet.java and script.js to update a status message on the home page each time the website is visited

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -13,7 +13,11 @@
 // limitations under the License.
 
 
-
+async function getPhrase() {
+  const response = await fetch('/data');
+  const phrase = await response.text();
```
Comment (lee12jin) [inline portfolio/src/main/webapp/script.js:18]: Please use let for local variables that get overwritten or changed. Const is generally for variables that get assigned once and never again.

https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -36,6 +36,15 @@ a{
   float:right;
 }
 
+#dynamicDescription{
```
Comment (lee12jin) [inline portfolio/src/main/webapp/style.css:39]: Stick to hyphenated css id/classes. 

https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

dynamic-description

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -21,6 +18,11 @@
     <div class="row">
 
      <div id="content">
+
+      <div id = "dynamicDescription">
```
Comment (lee12jin) [inline portfolio/src/main/webapp/index.html:22]: See below comment on hyphenated elements.

Plus, please keep consistent with the spacing
\<div id="dynamic-description">

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 4]
```diff
@@ -1,32 +0,0 @@
...
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
-public class DataServlet extends HttpServlet {
-
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
-  }
-}
```
Comment (lee12jin) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:32]: ?

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (nitinramadoss) [reply portfolio/src/main/webapp/script.js:18]: I'll keep that in mind, but for this implementation I did not change the values assigned to response and phrase after initializing them

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (nitinramadoss) [reply portfolio/src/main/webapp/style.css:39]: I will push the revision with that update now

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (nitinramadoss) [reply portfolio/src/main/webapp/index.html:22]: I'll fix the spacing and push a revision to the same branch

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 4 above)
Comment (nitinramadoss) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:32]: Oh sorry, I think I sent an older version of the DataServlet class I'll send the updated one now

File: portfolio/src/main/java/com/google/DataServlet.java [hunk 5]
```diff
@@ -13,20 +13,37 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+import java.util.ArrayList;
```
Comment (jasperlee-google) [inline portfolio/src/main/java/com/google/DataServlet.java:16]: Can you keep the imports in the ASCII sort order?
https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

File: portfolio/src/main/java/com/google/DataServlet.java [hunk 6]
```diff
@@ -13,20 +13,37 @@
...
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+@WebServlet("/data") 
 public class DataServlet extends HttpServlet {
 
+  private List<String> phrases;
+
+   @Override
```
Comment (jasperlee-google) [inline portfolio/src/main/java/com/google/DataServlet.java:31]: nit: looks like there is an extra space before this `@Override` statement.

File: portfolio/src/main/webapp/style.css [hunk 7]
```diff
@@ -36,6 +36,15 @@ a{
...
+  font-size: x-large;
+  font-family:Courier New;
```
Comment (jasperlee-google) [inline portfolio/src/main/webapp/style.css:41]: nit: think the spacing after the colon `:` should be consistent here so

`font-family: Courier New;`

Comment (nitinramadoss) [reply portfolio/src/main/webapp/style.css:52]: I'll fix that and push the update to the same branch

File: portfolio/src/main/java/com/google/DataServlet.java (hunk 6 above)
Comment (nitinramadoss) [reply portfolio/src/main/java/com/google/DataServlet.java:31]: Got it, I'll push the update

File: portfolio/src/main/java/com/google/DataServlet.java (hunk 5 above)
Comment (nitinramadoss) [reply portfolio/src/main/java/com/google/DataServlet.java:16]: Will do, I'll send the update to the same branch

File: portfolio/src/main/java/com/google/DataServlet.java [hunk 8]
```diff
@@ -11,22 +11,33 @@
...
 import javax.servlet.http.HttpServletResponse;
-
+ 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+@WebServlet("/data") 
 public class DataServlet extends HttpServlet {
-
+  private List<String> phrases;
+   @Override
```
Comment (lee12jin) [inline portfolio/src/main/java/com/google/DataServlet.java:29]: https://google.github.io/styleguide/javaguide.html#s4.6.1-vertical-whitespace
Put a line between each of your member variables and functions.

{
  private List ...

  @Override
  public void init() {
  }

  @Override
  public void doGet

Comment (nitinramadoss) [reply portfolio/src/main/java/com/google/DataServlet.java:29]: Thank you for approving, I have implemented this change for the next PR (not the jsonBranch but the postBranch that I will send today).

**Pull request merged**